framework_tvb/tvb/adapters/exporters/export_manager.py

```diff
@@ -135,7 +134,7 @@ def export_data(self, data, exporter_id, project):
         # This will imply to generate a folder which is unique for each export
         data_export_folder = None
         try:
-            data_export_folder = self._build_data_export_folder(data)
+            data_export_folder = self.storage_interface.build_data_export_folder(data, self.export_folder)
```

Comment:
Could this be computed inside tvb_stoarge? We would eliminate a parameter for the following storage_interface calls.

Reply:
What do you mean exactly by this?

Reply:
Wondering whether data_export_folder computation could move to tvb_storage